redhat-cop/infra-ansible
# Add custom managed IAM policies to the manage-aws-identities role

### What does this PR do?
Adds custom managed policies to the `identity-management/manage-aws-identities` role.

### How should this be tested?
Example inventories have been included in the README.md I have confirmed the add and delete work and that the policies will be correctly imported `from_yaml` and are functioning as expected with the newly created service account, policy and group by using a fairly restrictive route53 policy which only allows `list-resource-record-sets` and not `list-hosted-zones`

### Is there a relevant Issue open for this?
N/A

### Other Relevant info, PRs, etc.
N/A

### People to notify
cc: @redhat-cop/infra-ansible
